### PR TITLE
Fix some issues	with addServiceBusTopicConfig

### PR DESCRIPTION
**Pull request by Blackbaud-JasonBodnar**
* When adding connection strings in application-local.properties use the topic name to make the connection string specific to the topic. If the project previously had a service bus connection configured Spring can't load service bus beans because both service buses were using the same connection string

* Add missing imports for the payloadClassName and propertiesClassName to ComponentTestConfig.java

* In ComponentTestConfig when creating the ValidatingMessageHandler bean define	the method as the camel	case version of	the message handler class so it matches the @Qualifier used in the ServiceBusConsumer bean

* When adding a propertyWithSeparator don't add it if it already exists in the properties file

### CODE REVIEW

**Comment by Blackbaud-JasonBodnar** [comment]
@Blackbaud-MikeLueders 
@blackbaud/highlander

**Comment by Blackbaud-JasonBodnar** [comment]
@Blackbaud-BethRichardson conflicts have been fixed.

File: src/main/groovy/com/blackbaud/templates/project/AsyncProject.groovy [hunk 1]
```diff
@@ -148,9 +150,15 @@ class AsyncProject {
             addConsumerImports(componentTestConfigFile)
             componentTestConfigFile.addImport("com.blackbaud.azure.servicebus.consumer.handlers.ValidatingServiceBusMessageHandler")
             componentTestConfigFile.addImport("org.springframework.beans.factory.annotation.Qualifier")
+            componentTestConfigFile.addImport("${servicePackage}.api.${formatter.payloadClassName}")
+            componentTestConfigFile.addImport("${servicePackage}.servicebus.${formatter.propertiesClassName}")
             componentTestConfigFile.appendToClass("""
     @Bean
+<<<<<<< 08eb9a25280d004ff87894803e0ba0d1a4e11e8f
```
Comment (Blackbaud-BethRichardson) [inline src/main/groovy/com/blackbaud/templates/project/AsyncProject.groovy]: looks like some git comments left behind.

Comment (Blackbaud-JasonBodnar) [reply src/main/groovy/com/blackbaud/templates/project/AsyncProject.groovy]: Fixed.

I'm going through your other suggestions for .properties file changes so hold off on merging until I let you know.

**Comment by Blackbaud-JasonBodnar** [comment]
@Blackbaud-BethRichardson this is ready for re-review

File: src/componentTest/resources/expectedTemplateContent/service-bus-internal-topic-datasync/src/main/resources/application-vststest.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+servicebus.datasync.producer_connection_url=Endpoint=sb://namespace.servicebus.windows.net/;SharedAccessSignature=SharedAccessSignature sr=amqp%3A%2F%2Fnamespace.servicebus.windows.net%2Fdatasync&sig=token&skn=shared_access_profile_send // TODO: Create REX service bus topic and replace
+servicebus.datasync.consumer_connection_url=Endpoint=sb://namespace.servicebus.windows.net/;SharedAccessSignature=SharedAccessSignature sr=amqp%3A%2F%2Fnamespace.servicebus.windows.net%2Fdatasync%2Fsubscriptions%2Fsubscription_name&sig=token&se=1696020226&skn=shared_access_profile_listen // TODO:: Create REX service bus topic and replace
```
Comment (Blackbaud-BethRichardson) [inline src/componentTest/resources/expectedTemplateContent/service-bus-internal-topic-datasync/src/main/resources/application-vststest.properties]: extra colon. No biggie

File: src/componentTest/resources/expectedTemplateContent/service-bus-internal-topic-scheduled/src/main/resources/application-vststest.properties [hunk 3]
```diff
@@ -0,0 +1,2 @@
+servicebus.scheduled.producer_connection_url=Endpoint=sb://namespace.servicebus.windows.net/;SharedAccessSignature=SharedAccessSignature sr=amqp%3A%2F%2Fnamespace.servicebus.windows.net%2Fscheduled&sig=token&skn=shared_access_profile_send // TODO: Create REX service bus topic and replace
+servicebus.scheduled.consumer_connection_url=Endpoint=sb://namespace.servicebus.windows.net/;SharedAccessSignature=SharedAccessSignature sr=amqp%3A%2F%2Fnamespace.servicebus.windows.net%2Fscheduled%2Fsubscriptions%2Fsubscription_name&sig=token&se=1696020226&skn=shared_access_profile_listen // TODO:: Create REX service bus topic and replace
```
Comment (Blackbaud-BethRichardson) [inline src/componentTest/resources/expectedTemplateContent/service-bus-internal-topic-scheduled/src/main/resources/application-vststest.properties]: extra colon.

File: src/componentTest/resources/expectedTemplateContent/service-bus-internal-topic-datasync/src/main/resources/application-vststest.properties (hunk 2 above)
Comment (Blackbaud-JasonBodnar) [reply src/componentTest/resources/expectedTemplateContent/service-bus-internal-topic-datasync/src/main/resources/application-vststest.properties]: Fixed.

File: src/componentTest/resources/expectedTemplateContent/service-bus-internal-topic-scheduled/src/main/resources/application-vststest.properties (hunk 3 above)
Comment (Blackbaud-JasonBodnar) [reply src/componentTest/resources/expectedTemplateContent/service-bus-internal-topic-scheduled/src/main/resources/application-vststest.properties]: Fixed.

**Review by Blackbaud-BethRichardson — APPROVED** [review]

**Pull request merged**